src/index.js

```diff
@@ -2,22 +2,24 @@ import { GraphQLServer } from 'graphql-yoga';
 import { Prisma } from 'prisma-binding';
 import { resolvers } from './resolvers';
 
+const { PRISMA_SERVER_DOMAIN, PRISMA_SERVER_PORT, DEBUG, SERVER_PORT, PRISMA_SECRET } = process.env;
```

Comment:
You can make a configuration folder where you can get all environment variables and then use in every file importing the configuration instead of catching process.env everywhere